### ISSUE TITLE
1.21 

### ISSUE THREAD

**Issue by TheKerbecs** [RohanFredriksson/minecraft-server-starter#1]
Hi, will it be updated to 1.21?

**Comment by RohanFredriksson** [comment]
Hi @TheKerbecs, unfortunately I can't update to 1.21 at this time as the code is heavily reliant on the Node Minecraft Protocol. Once there is a new release that supports the 1.21 version, I should be able to make some changes to make it work. Thanks

**Comment by RohanFredriksson** [comment]
Hey @TheKerbecs, I've managed to update this project to work with 1.21 through a complete code rewrite. Actually, this project now works for all versions 1.7 and above and should work for all future versions of Minecraft as long as Mojang doesn't change the login protocol (which they haven't done in 11 years). Hopefully you'll find this useful.

**Comment by TheKerbecs** [comment]
Hey, that's amazing! Thank you so much :) I hope more people will find this repo.

**Comment by TheKerbecs** [comment]
@RohanFredriksson I tried to run the project, when I followed the instructions the Python file ran but the server says: Can't connect to the server, when I tried to connect the error: getsockopt appeared and the server didn't start (on the 25565 port). 
In the console it says Serving sleeping. Waiting for login attempt...
The server does start as expected when I run the server file and connect with port 25567. Any idea what can cause this issue?

Edit:
It works when I run it on my computer and connect through localhost but it doesn't work on my raspberry pi, which is on the same network.
